toolchain/check/eval.cpp

```diff
@@ -391,6 +391,9 @@ static auto PerformBuiltinUnaryIntOp(Context& context, SemIRLoc loc,
       }
       op_val.negate();
       break;
+    case SemIR::BuiltinFunctionKind::IntUNegate:
+      op_val.negate();
+      break;
```

Comment:
Do unsigned types implement negate?

Reply:
Per [the design](https://github.com/carbon-language/carbon-lang/blob/trunk/docs/design/expressions/arithmetic.md#integer-types:~:text=Signed%20and%20unsigned%20integer%20types%20support%20all%20the%20arithmetic%20operators), yes. I'm not sure we explicitly considered the alternative, but we probably would find use for a builtin for a wrapping negate even if we don't expose it as a unary `-` operator.